fdbclient/FDBTypes.h

```diff
@@ -1126,4 +1126,27 @@ inline const char* transactionPriorityToString(TransactionPriority priority, boo
 	throw internal_error();
 }
 
+struct VersionVector {
+	std::map<Tag, Version> versions;
+
+	VersionVector() {}
+
+	void setVersion(const Tag& tag, Version version) {
+		versions[tag] = version;
```

Comment:
can assert `tag != invalidTag` here and in `hasVersion`

Reply:
Good suggestion. Done.